framework/py/flwr/common/inflatable.py

```diff
@@ -31,14 +30,20 @@ def deflate(self) -> bytes:
         raise NotImplementedError()
 
     @classmethod
-    def inflate(cls, object_content: bytes) -> "InflatableObject":
+    def inflate(
+        cls, object_content: bytes, children: dict[str, "InflatableObject"]
+    ) -> "InflatableObject":
         """Inflate the object from bytes.
 
         Parameters
         ----------
         object_content : bytes
             The deflated object content.
 
+        children : dict[str, InflatableObject]
+            Dict of children InflatableObjects mapped to thier Object ID. These
+            children enable the full inflation of the parent InflatableObject.
```

Comment:
For consistency, we'd avoid using shorthand "Dict" for `Dictionary`.

```suggestion
            Dictionary of children InflatableObjects mapped to thier Object ID. These
            children enable the full inflation of the parent InflatableObject.
```